frontend/integration-tests/views/secrets.view.ts

```diff
@@ -77,3 +97,34 @@ export const editSecret = async(ns: string, name: string, updateForm: Function)
   await saveButton.click();
   expect(crudView.errorMessage.isPresent()).toBe(false);
 };
+
+export const addSecretToWorkloadAsEnv = async(workloadname: string, EnvPrefix: string) => {
+  await clickAddSecretToWorkload();
+  await browser.wait(until.presenceOf(selectWorkloadBtn));
+  await selectWorkloadBtn.click();
+  await browser.wait(until.presenceOf($('li[role=option]')), 5000);
+  await element(by.cssContainingText('li[role=option] a', workloadname)).click();
+  await addSecretAsEnv.click();
+  await addSecretAsEnvPrefixInput.sendKeys(EnvPrefix);
+  await monitoringView.modalConfirmButton.click();
+};
+
+export const addSecretToWorkloadAsVol = async(workloadname: string, MntPath: string) => {
+  await clickAddSecretToWorkload();
+  await selectWorkloadBtn.click();
+  await browser.wait(until.presenceOf($('li[role=option]')), 5000);
+  await element(by.cssContainingText('li[role=option] a', workloadname)).click();
+  await addSecretAsVol.click();
+  await addSecretAsVolMntPathInput.sendKeys(MntPath);
+  await monitoringView.modalConfirmButton.click();
+};
+
+export function getResourceJSON(name: string, namespace: string, kind: string) {
+  return execSync(`kubectl get -o json -n ${namespace} ${kind} ${name}`).toString();
+}
+
+export function searchJSON(propertyPath: string, value, name: string, namespace: string, kind: string):boolean {
```

Comment:
Sometimes we need call `searchJSON` to get a number, so removed the type of parameter `value` , I'm not sure if it's the right way do that